sdk-api-src/content/fileapi/nf-fileapi-setfiletime.md

```diff
@@ -94,12 +94,6 @@ To prevent file operations using the given handle from modifying the last access
 A pointer to a <a href="/windows/desktop/api/minwinbase/ns-minwinbase-filetime">FILETIME</a> structure that 
        contains the new last modified date and time for the file or directory.  If the application does not need to change this information, set this parameter either  to <b>NULL</b>  or to a pointer to a <b>FILETIME</b> structure that has both the <b>dwLowDateTime</b> and <b>dwHighDateTime</b> members set to 0.
 
-To prevent file operations using the given handle from modifying the last access time, call 
```

Comment:
I checked with the product team, and this paragraph should be kept. We should just update "last access time" to "last write time" in this existing paragraph.